### PR TITLE
In deploy instructions, recommend creating a new project

### PR DESCRIPTION
**Pull request by melissachang**
This is safer because there will be fewer Editor/Owners, and it will be easier to verify that all Editor/Owners already have access to BigQuery tables.

### CODE REVIEW

File: bigquery/deploy/README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
+    * Create a project for deploying Data Explorer. We recommend creating a
+      project because all project Editors/Owners will indirectly have
+      access to the BigQuery tables. (Project Editors/Owners by default have
+      permission to act as service accounts, and the indexer service account will be
+      given permission to read the BigQuery tables.)
+      * Ensure that anyone who is
+      granted Editor/Owner roles in this project, already has access to the BigQuery tables.
     * Create the service account
       * Navigate to `IAM & Admin > Service Accounts > Create Service Account`.
       * We suggest the name `DATASET-data-explorer-indexer` to make it clear
```
Comment (alahwa) [inline bigquery/deploy/README.md]: Below here, mention that the Storage Object Viewer role is for accessing GCS, including GCR where the docker image is stored (in case anybody has sensitive data in GCS they're trying to limit access to)

File: bigquery/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ### Quickstart
 
+Index the default [public BigQuery table](https://bigquery.cloud.google.com/table/google.com:biggene:platinum_genomes.sample_info) into a local Elasticsearch container.
```
[the remark this answers is not in the document]
Comment (melissachang) [reply bigquery/README.md]: Done

File: bigquery/deploy/README.md [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
-    * [Following the principle of least privilege](https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform#why_use_service_accounts),
-    we create a service account and give it only the necessary permissions,
-    rather than use the default Compute Engine service account. At the end of
-    this section, the new service account will have access to the BigQuery
-    tables for your dataset. Before creating the service account, you must
-    ensure that everyone who could use this service account already has access
-    to the BigQuery tables. To see who could use this service account, navigate
-    to `IAM & Admin` and look for `Owner`, `Editor`, and `Service Account User`
-    roles.
+  dataset
```
[the remark this answers is not in the document]
Comment (melissachang) [reply bigquery/deploy/README.md]: Needed to force "Following the principle" to next line

File: bigquery/deploy/README.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
-    this section, the new service account will have access to the BigQuery
-    tables for your dataset. Before creating the service account, you must
-    ensure that everyone who could use this service account already has access
-    to the BigQuery tables. To see who could use this service account, navigate
-    to `IAM & Admin` and look for `Owner`, `Editor`, and `Service Account User`
-    roles.
+  dataset  
+  [Following the principle of least privilege](https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform#why_use_service_accounts),
+  we recommend using a service account with only the necessary permissions,
+  rather than the default Compute Engine service account.
```
[the remark this answers is not in the document]
Comment (melissachang) [reply bigquery/deploy/README.md]: Don't think it's needed, think it's too much detail

File: bigquery/deploy/README.md (hunk 1 above)
Comment (melissachang) [reply bigquery/deploy/README.md]: Done

File: bigquery/deploy/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (alahwa) [reply bigquery/deploy/README.md]: I think markdown doesn't care where the whitespace goes, but by convention it'll either be an additional newline or whitespace at the beginning of the next line.

File: bigquery/deploy/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (alahwa) [reply bigquery/deploy/README.md]: Without knowing that the default account comes with Editor permissions, it's not clear to me what principle of least privilege has to do with using a service account other than the default.

File: bigquery/deploy/README.md [hunk 5]
```diff
@@ -2,17 +2,17 @@
...
+    * Create a project for deploying Data Explorer. We recommend creating a
```
Comment (alahwa) [inline bigquery/deploy/README.md]: This would be a separate PR, but if we recommend creating a new project, that should address your concern with reserving a static internal IP address for the Elasticsearch client so we can eliminate a bunch of config/steps both here and in deploying data explorer, right?

File: bigquery/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alahwa) [reply bigquery/README.md]: To clarify, I meant a newline after the link and before `into` to prevent this line from being ridiculously long for readability purposes.

File: bigquery/deploy/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (melissachang) [reply bigquery/deploy/README.md]: Neither of those work. This is known behavior of GitHub markdown. Search https://github.github.com/gfm/ for "two spaces". You can try for yourself in atom.

File: bigquery/deploy/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (melissachang) [reply bigquery/deploy/README.md]: Done

File: bigquery/deploy/README.md (hunk 5 above)
Comment (melissachang) [reply bigquery/deploy/README.md]: I still don't think we should use static IP for the reasons discussed earlier, let's discuss ofline.

Eliminating the bunch of config/steps can easily be done with a shell script, as I described yesterday. I hope to start this in the next week or so, you're welcome to try too.

File: bigquery/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (melissachang) [reply bigquery/README.md]: Isn't that how it is now?
